en/frames_vtol/vtol_quadplane_foxtech_loong_2160.md

```diff
@@ -0,0 +1,369 @@
+   :::
+
+#### Lidar
+
+1. Mark the location to install the lidar with some tape or a pen.
+   Cut a hole inside the PVC shell and the foam, so that the lidar fits in place.
+
+   ![Prepared lidar hole](../../assets/airframes/vtol/foxtech_loong_2160/12-lidar-01.jpg)
+
+1. Secure the lidar with hot glue.
+
+   ![Installed lidar](../../assets/airframes/vtol/foxtech_loong_2160/13-lidar-02.jpg)
+
+#### GPS/Compass
+
+1. Use double sided tape to mount the GPS in the rear of the vehicle underneath the rear latch.
+
+   ![Installed GPS](../../assets/airframes//vtol/foxtech_loong_2160/14-gps.jpg)
+
+   The arrow on the GPS for the orientation can be ignored.
+   The orientation will be detected by the flight controller during the calibration.
+
+### Flight Controller
+
+Install either the Pixhawk or Skynode onto the baseplate.
+
+#### Pixhawk 6c/6c mini
+
+1. Use double sided tape to mount the flight controller to the baseplate.
+
+#### Skynode
+
+1. Use 4x M3x8 screws to mount the Skynode to the baseplate.
+   Make sure that the top of the "A" is facing to the front of the vehicle.
+1. Plug the 40A Power Module into the upper one of the two power connectors.
+1. Plug one (or if needed two) USB adapters into the 4-pin JST-GH connectors into the back of the Skynode and feed them to the front of the plate.
+   Fix the cables with zip ties in place.
+1. Tape a I2C splitter to the front right side of the baseplate (The splitter can be used to plug in ETH devices such as a radio link.)
+1. Connect the I2C splitter with the ETH port in the back of the Skynode.
+1. Plug in the two 40-pin cables into the front of the Skynode.
+1. Plug in the USB-C extension cable and bend it over to the front.
+   The bend needs to be very tight, so that the plate will fit into the vehicle.
+
+![Installed Skynode](../../assets/airframes/vtol/foxtech_loong_2160/15-skynode.jpg)
+
+#### Adapter Boards
+
+1. Screw the Pixhawk adapter Boards to the top plate.
+
+### Antennas and RC Receiver
+
+1. Tape the Skynode LTE antennas to the side of the fuselage as shown in the picture:
+
+   ![LTE-Antennas](../../assets/airframes/vtol/foxtech_loong_2160/16-lte-antennas.jpg)
+
+1. If you are using a radio telemetry module you can mount the antennas to the top of the fuselage.
+   In the front you can mount the antenna extension cable directly.
+
+   ![WIFI-Antennas-Front](../../assets/airframes/vtol/foxtech_loong_2160/17-antenna-front.jpg)
+
+   In the back you can use the 3D-Printed antenna adapter. The adapter can be glued in place with hot glue.
+
+   ![WIFI-Antenna-Back]()
+
+### 12S Power Module
+
+The 12S Power Module will be mounted on top of the battery.
+Plug the XT90 that is installed inside the vehicle into the PM.
+The power cable to connect the Skynode needs to be extended.
+This is necessary to get battery readings from the power module.
+
+The power module can be used as a 5V backup for Skynode.

```

Comment:
1. Good to have an image of it mounted.
2. This is a second 12S power module right? Or is it the same 40A PM you refer to in a previous section? If they are different good to explain when you need both.